docs/TutorialWebpack.md

```diff
@@ -59,25 +61,32 @@ Since these files aren't particularly useful in tests anyway we can safely mock
 And the mock file's themselves:
 
 ```js
-// test/styleMock.js
+// __mocks__/styleMock.js
 
-// Return an object to emulate css modules (if you are using them)
 module.exports = {};
 ```
 
-Alternatively, you can use an [ES2015 Proxy](https://github.com/keyanzhang/identity-obj-proxy)
-(`npm install --save-dev identity-obj-proxy` or create your own) to mock [CSS Modules](https://github.com/css-modules/css-modules); then all your className
+```js
+// __mocks__/fileMock.js
+
+module.exports = 'test-file-stub';
+```
+
+For CSS Modules, you can use an [ES2015 Proxy](https://github.com/keyanzhang/identity-obj-proxy)
+(`npm install --save-dev identity-obj-proxy`) to mock
+[CSS Modules](https://github.com/css-modules/css-modules); then all your className
 lookups on the styles object will be returned as-is (e.g., `styles.foobar === 'foobar'`).
+This is pretty handy for React snapshot testing.
 
 Notice that Proxy is enabled in Node.js `v6.*` by default; if you are not on Node `v6.*` yet,
 make sure you invoke Jest using `node --harmony_proxies node_modules/.bin/jest`.
 
 ```js
-// package.json
+// package.json (for CSS Modules)
 {
   "jest": {
     "moduleNameMapper": {
-       "^.+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "identity-obj-proxy",
+       "^.+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
       "^.+\\.(css|less)$": "identity-obj-proxy"
```

Comment:
It's fine for css files (also see https://github.com/keyanzhang/jest-css-modules-example/blob/master/package.json#L51-L52). However, if you mock both static assets and css files to the proxy, and have a component that imports both types it crashes. Not exactly sure why...